### PR TITLE
updated the members list (as of October 11)

### LINKED ISSUE #39: Add data to team.json file and images for post bearers.
- [ ] Get club members info [here](https://drive.google.com/open?id=1VUwb1rycg2L5a0ge68JbApR_1dV4Ios0) and convert it into JSON using the google sheets add-on.
- [ ] All the profile links of members to be updated.
- [ ] Post bearer's images to be added in images/posts/ directory and the same to be updated in team.json.

### PR DESCRIPTION
**Pull request by prajwal27**
Closes #39

### CODE REVIEW

File: data/team.json [hunk 1]
```diff
@@ -1,812 +1,779 @@
...
-		"Email Address": "email@gmail.com",
-		"Github": "https://github.com/",
-		"Linkedin": "https://www.linkedin.com/",
-		"Image": "images/posts/default.jpg",
-		"Post": "HakerRank Chapter Head",
-	},
+        {
+            "name": "Vilas Bhat",
+            "post": "Convener",
+            "imageUrl": "images/posts/default.jpg"
```
Comment (Hrily) [inline data/team.json:5]: I think we have image urls in excel sheet

File: data/team.json [hunk 2]
```diff
@@ -1,812 +1,779 @@
...
-		"Github": "https://github.com/",
-		"Linkedin":"https://www.linkedin.com/"
-	},
-	{
-		"Name": "Vinay Bhat",
-		"Email Address": "vinaybhat001@gmail.com",
-		"Github": "https://github.com/vshan",
-		"Linkedin":"https://www.linkedin.com/vshan"
-	}
+            "name": "Adarsh Honawad"
```
Comment (Hrily) [inline data/team.json:128]: The details of these people?

**Review by Hrily — CHANGES_REQUESTED** [review]

File: data/team.json (hunk 1 above)
Comment (prajwal27) [reply data/team.json:5]: ImageUrl is just a variable. If they have submitted it through excel sheet , then we'll be updating it weekly, else imageUrl contains the default image

File: data/team.json (hunk 2 above)
Comment (prajwal27) [reply data/team.json:128]: These people have not yet submitted their details in the excel sheet.

**Review by prajwal27 — COMMENTED** [review]
The members' details is shown only if they have updated it in the google form.

**Comment by sbshah97** [comment]
@hrily can we link the Google Sheet and the data that we fetch from it

**Comment by sbshah97** [comment]
https://coderwall.com/p/duapqq/use-a-google-spreadsheet-as-your-json-backend

**Comment by sbshah97** [comment]
@prajwal27 any updates?

**Pull request closed without merging**